components/edit-collective/EditTransferWiseAccount.js

```diff
@@ -92,7 +74,19 @@ const EditTransferWiseAccount = props => {
     return (
       <form onSubmit={formik.handleSubmit}>
         <P lineHeight="0" fontSize="Caption" color="black.600" fontWeight="normal">
-          {props.intl.formatMessage(messages.description)}
+          <FormattedMessage
+            id="collective.create.connectedAccounts.transferwise.description"
+            defaultMessage="Connect a TransferWise account to pay expenses with one click. For instructions on how to connect to TrasnferWise, please, <a>read our documentation</a>."
+            values={{
+              a(...chunks) {
+                return (
+                  <a href="https://docs.opencollective.com/help/fiscal-hosts/payouts/payouts-with-transferwise">
+                    {chunks}
+                  </a>
+                );
+              },
+            }}
```

Comment:
`I18nFormatters` provides an easier way to achieve that. I'd also suggest opening the link in a new tab

```suggestion
            values={{
              a: getI18nLink({
                href: 'https://docs.opencollective.com/help/fiscal-hosts/payouts/payouts-with-transferwise',
                openInNewTab: true,
              })
            }}
```